crmsh/utils.py

```diff
@@ -2649,4 +2649,46 @@ def ping_node(node):
     rc, _, err = get_stdout_stderr("ping -c 1 {}".format(node))
     if rc != 0:
         raise ValueError("host \"{}\" is unreachable: {}".format(node, err))
+
+
+def is_quorate(expected_votes, actual_votes):
+    """
+    Given expected votes and actual votes, calculate if is quorated
+    """
+    return int(actual_votes)/int(expected_votes) > 0.5
+
+
+def get_stdout_or_raise_error(cmd):
+    """
+    Common function to get stdout from cmd or raise exception
+    """
+    rc, out, err = get_stdout_stderr(cmd)
+    if rc != 0:
+        raise ValueError("Failed to run \"{}\": {}".format(cmd, err))
+    return out
+
+
+def get_quorum_votes_dict():
+    """
+    Get current expect votes and total votes
```

Comment:
A would add here in the docstring the value of the returned dictionaly. Otherwise, we cannot really undestand what it is returning

Reply:
Changed